README.md

```diff
@@ -21,6 +21,9 @@ We can likely swap to terraform once it's supported there, but currently there a
 
 ## Operational concerns
 
+### Graph Permissions
```

Comment:
this is in the wrong section it should be in adding a new application, not in operational concerns

Reply:
Oops that's where I thought I'd added - will move